cnf-certification-test/accesscontrol/suite.go

```diff
@@ -293,7 +293,8 @@ func TestNamespace(env *provider.TestEnvironment) {
 		ginkgo.Fail("error retrieving CRs")
 	}
 
-	invalidCrsNum := namespace.GetInvalidCRsNum(invalidCrs)
+	invalidCrsNum, claimsLog := namespace.GetInvalidCRsNum(invalidCrs)
+	tnf.ClaimFilePrintf("%s", claimsLog)
```

Comment:
Normally, the claim file log output is only helpful when the tc fails. I'd move this printf inside the `if invalidCrsNum > 0` block.

Reply:
Fixed!